app/controllers/miq_policy_controller.rb

```diff
@@ -943,7 +943,9 @@ def folder_get_info(folder_node)
     @sb[:folder] = nodeid.nil? ? nodetype.split("-").last : nodeid
     if x_active_tree == :policy_tree
       if nodeid.nil? && ["compliance", "control"].include?(nodetype.split('-').last)
-        @folders = ["Host #{nodetype.split('-').last.titleize}", "Vm #{nodetype.split('-').last.titleize}"]
+        @folders = MiqPolicy::UI_FOLDERS.collect do |model|
```

Comment:
Yeah, @moolitayer, I agree.  We can tackle this later.  It's really what folders we see in the UI for MiqPolicy so who knows where it should really live but the need to format these classes for view purposes definitely feels like it should be a thing that lives elsewhere at some point.